components/vf-form__checkbox/vf-form__checkbox.scss

```diff
@@ -9,19 +9,19 @@
  * Location: #{map-get($componentInfo, 'location')}
  */
 
-.vf-form__item.vf-form__item--checkbox {
-  margin-bottom: 16px;
-}
-
 // Because we want the checkbox to be used in various context. We need the context to define
 // Any layout things, and colours like below
 .vf-form__item--checkbox {
   .vf-form__checkbox {
     + .vf-form__label::before {
-      // border-color: ui-color(grey--light);
+      // border-color: set-ui-color(vf-ui-color--grey--light);
       margin-right: 8px;
     }
   }
+
+  .vf-form__helper {
+    margin-left: 29px; // just because
```

Comment:
best comment I've seen in a while

Reply:
I think it boils down to the line-height for components taking the line height for long form content that mark created -- so it's all a little off

need to add that 'leading-trim' CSS hack